python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -930,21 +990,62 @@ def run_transformer_autocast_bf16( repo_id,
     today = date.today()
     if 'exclude' in conf:
         excludes = conf['exclude']
-    
+    if conf['plot_memory_vs_time_graph'] == True:
+        process = subprocess.Popen(['top', '-b', '-n', '1'], stdout=subprocess.PIPE, text=True)
```

Comment:
This may not work on Windows

Reply:
Yes, it only works on Linux now. And I mention it in README file and set the defalut parameter to False.